### PR TITLE
onet.pl share icon fix

### PR DESCRIPTION
**Pull request by FadeMind**
https://www.onet.pl/informacje/onetwiadomosci/koronawirus-zmiany-w-obostrzeniach-rzecznik-rzadu-i-wiceminister-komentuja/xghcbcx,79cfc278

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -10438,21 +10438,23 @@ onet.pl
 plejada.pl
 
 INVERT
-.websiteLogo
-.serviceIcon
-a.serviceLogo > img
-.sheet
-.logoImage
+.WeatherDay_tempIcon__3Uzdm
+._3ZySwSLi_pur0unnAQO2No
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10442]: Could this be changed to a more generic selector, which doesn't have the random characters.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -10438,21 +10438,23 @@ onet.pl
 plejada.pl
 
 INVERT
-.websiteLogo
-.serviceIcon
-a.serviceLogo > img
-.sheet
-.logoImage
+.WeatherDay_tempIcon__3Uzdm
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10441]: Ditto, 
```suggestion
[class*="WeatherDay_tempIcon"]
```

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:10442]: See https://github.com/darkreader/darkreader/pull/2021/files

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:10441]: https://github.com/darkreader/darkreader/pull/4154/files

**Review by Gusted — APPROVED** [review]

**Pull request merged**